ui/pages/reservation-unit/[...params].tsx

```diff
@@ -600,19 +600,21 @@ const ReservationUnitReservation = ({
             <FormProvider {...form}>
               <div>
                 <Title>{pageTitle}</Title>
-                <StyledStepper
-                  language={i18n.language}
-                  selectedStep={step}
-                  small={steps.length > 2}
-                  onStepClick={(e) => {
-                    const target = e.currentTarget;
-                    const s = target
-                      .getAttribute("data-testid")
-                      .replace("hds-stepper-step-", "");
-                    setStep(parseInt(s, 10));
-                  }}
-                  steps={steps}
-                />
+                {steps.length <= 2 && (
```

Comment:
This reads like only show the stepper if steps <= 2. 2 steps is a free reservation?
So only show stepper for free reservations?

Reply:
Oh crap, the title falsely says otherwise. So the objective was to hide it for _paid_ reservations.